wordpress-mobile/WordPress-Android
# Reenables support for Global Style Colors with Full Site Editing Themes

## Description:
This enables Global Style Support for FSE sites introduced with https://github.com/wordpress-mobile/WordPress-Android/pull/15050 and temporarily disabled with https://github.com/wordpress-mobile/WordPress-Android/pull/15058.
A full clean up of the configuration will be performed with https://github.com/wordpress-mobile/WordPress-Android/issues/15132 

To test:
Check instructions in https://github.com/wordpress-mobile/WordPress-Android/pull/15050

## Regression Notes
1. Potential unintended areas of impact
This is making the feature added in https://github.com/wordpress-mobile/WordPress-Android/pull/15002 available to everyone.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
Verify that the existing test pass

PR submission checklist:
